R/make_catboost.R

```diff
@@ -0,0 +1,143 @@
+#' Automated function for explaining catboost models
+#' 
+#' Function \code{make_catboost} creates catboost model with default parameters using ranger library and
+#' wraps it with the explainer.
+#' 
+#' @param data training set for ranger model. It can be data frame, matrix, data.table or dgCMatrix. Note: data should contain target column
+#' @param target name of a target value. Should be character. Has to be in data frame
+#' @param type specify weather it is classification task or regression. Should be one of these characters: "classif", "regr"
```

Comment:
`classif `or `classification`
`regr` or `regression`
Please change to one verison in all function.